common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -236,10 +236,13 @@ object ContentWidths {
 
   object GalleryMedia {
     val inline = WidthsByBreakpoint(
-      mobile          = Some(445.px),
-      mobileLandscape = Some(610.px),
-      phablet =         Some(620.px),
-      tablet =          Some(700.px)) // desktop, leftCol, and wide are also 700px
+      mobile          = Some(480.px),
+      mobileLandscape = Some(660.px),
+      phablet =         Some(700.px),
+      tablet =          Some(700.px), // TODO: Change to 480 when new galleries is merged
+      desktop =         Some(720.px),
+      leftCol =         Some(880.px),
+      wide =            Some(1010.px))
```

Comment:
So it's almost always scaled. Sometimes up, sometimes down... For fixed size portrait images, @rich-nguyen's plan would work: https://github.com/guardian/frontend/issues/12216#issuecomment-196869603, but for the fluid ones, we would have to have a clever mechanism, similar to what @gtrufitt mentioned re:immersives (https://github.com/guardian/frontend/pull/12775#issuecomment-216855041), so that only _meaningful_ viewport height changes initiates a new request... That's quite a bit of work, no? (I will be 👀 , though).

What about the simple case of landscapes shown at different width to requests - maybe at least this can be fixed before the rollout?

P.s. I like how it works at https://samefourchords.com/! But there are too many new requests for my liking.

![too-many-requests](https://cloud.githubusercontent.com/assets/6032869/15069423/564b31d0-1375-11e6-92db-92e98b27f20b.png)

I would adjust the logic, so that the smaller images (for when the viewport's height shrinks) are not new requests, but downsized, already downloaded ones. I would also initiate a higher res uploads at looser steps, maybe. As a last point I would love for the page (image in view) to always stay put and not disappear from view when you change viewport height (orientation). See?! Now I'm commenting on your private projects , oh gawd! 🙀  Zakopane's cool, if a tad too touristy...